docs/sources/reference/run.md

```diff
@@ -134,6 +134,11 @@ While not strictly a means of identifying a container, you can specify a version
 image you'd like to run the container with by adding `image[:tag]` to the command. For
 example, `docker run ubuntu:14.04`.
 
+### Image[@digest]
+
+Images using the v2 or later image format may be referred to using their
```

Comment:
Images that use the v2 or later format, have a content-addressable identifiers called a _digest_. As long as the input used to generate the image is unchanged, the digest value is predictable and referenceable.